packages/expect/src/index.ts

```diff
@@ -389,6 +393,10 @@ expect.any = any;
 expect.not = {
   arrayContaining: arrayNotContaining,
   closeTo: notCloseTo,
+  numberGreaterThan,
+  numberGreaterThanOrEqualTo,
+  numberLessThan,
+  numberLessThanOrEqualTo,
```

Comment:
Are these doing the right job? I mean, will `expect.not.numberLessThan(10)` be passing only with numbers larger than 10? Also this assertion would be somewhat similar to `expect.numberGreaterThan(10)`. So, is there a need to have `.not` matchers at all in this case?

Reply:
I think you're right. I added them here only to follow the current API, as most of the matchers are present on the `expect.not`, I thought it would be a good idea to add these too. 
Should I remove them?